src/engine/platform/msm6295.cpp

```diff
@@ -206,7 +206,7 @@ int DivPlatformMSM6295::dispatch(DivCommand c) {
 
 void DivPlatformMSM6295::muteChannel(int ch, bool mute) {
   isMuted[ch]=mute;
-  msm.m_voice[ch].m_muted=mute;
+  msm.voice_mute(ch,mute);
```

Comment:
Notice: mute state may be lost on reset. Please confirm that this isn't the case.

Reply:
Yeah, mute state is lost.... Pushing a fix.